tools/explorer/lib/model.py

```diff
@@ -112,7 +114,9 @@ def __eq__(self, other):
 
   def __str__(self):
     standard_str = super().__str__()
-    return standard_str + '_' + self._increment + ': ' + str(self._is_optional)
+    if self._is_optional:
+      return standard_str + '_' + self._increment + ': optional'
+    return standard_str + '_' + self._increment + ': required'
```

Comment:
nit (also i think the g3 linter will yell about preferring f-string formatting):

```
optional_value = 'optional' if self._is_optional else 'required'
return f'{standard_str}_{self._increment}: {optional_value}'
```

Reply:
Done.